Tests/GRPCHTTP2TransportTests/HTTP2TransportNIOPosixTests.swift

```diff
@@ -159,4 +164,165 @@ final class HTTP2TransportNIOPosixTests: XCTestCase {
       }
     }
   }
+
+  #if canImport(NIOSSL)
+  static let samplePemCert = """
```

Comment:
Does it matter – for our tests – if this expires? If so then we should make it obvious when they are expired and easy to update them in an automated way.

Reply:
I don't think `NIOSSL/TLSConfiguration` do any validations at config creation time about the validity of the certificate.